node/pkg/governor/generated_mainnet_tokens.go

```diff
@@ -1,1287 +1,1411 @@
+		{chain: 2, addr: "000000000000000000000000595832f8fc6bf59c85c527fec3740a1b7a361269", symbol: "POWR", coinGeckoId: "power-ledger", decimals: 6, price: 0.355901},                                           // Addr: 0x595832f8fc6bf59c85c527fec3740a1b7a361269, Notional: 3381.0595000000003
+		{chain: 2, addr: "00000000000000000000000022c5543d1a35178cb03b33f929a959145e538532", symbol: "WAL", coinGeckoId: "the-wasted-lands", decimals: 18, price: 0.00030303},                                     // Addr: 0x22c5543d1a35178cb03b33f929a959145e538532, Notional: 0.39996680516612815
+		{chain: 2, addr: "00000000000000000000000032353a6c91143bfd6c7d363b546e62a9a2489a20", symbol: "AGLD", coinGeckoId: "adventure-gold", decimals: 18, price: 1.61},                                            // Addr: 0x32353a6c91143bfd6c7d363b546e62a9a2489a20, Notional: 12.1104588654
+		{chain: 2, addr: "00000000000000000000000056b4f8c39e07d4d5d91692acf9d0f6d4d3493763", symbol: "TRISM", coinGeckoId: "trism", decimals: 18, price: 0.00052052},                                              // Addr: 0x56b4f8c39e07d4d5d91692acf9d0f6d4d3493763, Notional: 8.681387859739393
+		{chain: 2, addr: "000000000000000000000000f2fdd9c25d7bc8002ce89716d1be484b2d976944", symbol: "𝕏PAY", coinGeckoId: "payments", decimals: 18, price: 0.00088013},                                            // Addr: 0xf2fdd9c25d7bc8002ce89716d1be484b2d976944, Notional: 22.00325
+		{chain: 2, addr: "0000000000000000000000009625ce7753ace1fa1865a47aae2c5c2ce4418569", symbol: "KAP", coinGeckoId: "kapital-dao", decimals: 18, price: 0.125837},                                            // Addr: 0x9625ce7753ace1fa1865a47aae2c5c2ce4418569, Notional: 1258.3700000000001
+		{chain: 2, addr: "0000000000000000000000000316eb71485b0ab14103307bf65a021042c6d380", symbol: "HBTC", coinGeckoId: "huobi-btc", decimals: 18, price: 23302},                                                // Addr: 0x0316eb71485b0ab14103307bf65a021042c6d380, Notional: 2368.1449767999998
+		{chain: 2, addr: "0000000000000000000000006911f552842236bd9e8ea8ddbb3fb414e2c5fa9d", symbol: "SNP", coinGeckoId: "synapse-network", decimals: 18, price: 0.00531536},                                      // Addr: 0x6911f552842236bd9e8ea8ddbb3fb414e2c5fa9d, Notional: 21.5857098436341
+		{chain: 2, addr: "000000000000000000000000ea068fba19ce95f12d252ad8cb2939225c4ea02d", symbol: "FIEF", coinGeckoId: "fief", decimals: 18, price: 0.01725023},                                                // Addr: 0xea068fba19ce95f12d252ad8cb2939225c4ea02d, Notional: 135773.47387824807
+		{chain: 2, addr: "000000000000000000000000eb4c2781e4eba804ce9a9803c67d0893436bb27d", symbol: "renBTC", coinGeckoId: "renbtc", decimals: 8, price: 6980.54},                                                // Addr: 0xeb4c2781e4eba804ce9a9803c67d0893436bb27d, Notional: 6.98054
```

Comment:
Hey @djb15 are you sure this one for renBTC is correct?

```json
 {
  "token": "2-000000000000000000000000eb4c2781e4eba804ce9a9803c67d0893436bb27d-renBTC",
  "previousPrice": 51499,
  "newPrice": 6980.54,
  "percentageChange": "-86.4"
 },
```

Coinmarketcap [has renBTC](https://coinmarketcap.com/currencies/renbtc/) at $65,708.81 as of right meow and the [etherscan token tracker for renBTC](https://etherscan.io/token/0xeb4c2781e4eba804ce9a9803c67d0893436bb27d) has it at ~$66k right now.

The `newPrice` being $6,980 seems off to me sir.

Reply:
I regenerated the token list and now it's at a more sensible price again